inf-clojure.el

```diff
@@ -876,6 +943,46 @@ to suppress the usage of the target buffer discovery logic."
     (inf-clojure (inf-clojure-cmd (inf-clojure-project-type)))
     (rename-buffer target-buffer-name)))
 
-(provide 'inf-clojure)
+;;;; Lumo
+;;;; ====
+
+;; When we'll have tests, this is one
+;; (let ((form "(js/global.hasOwnProperty \"$$LUMO_GLOBALS\")")
+;;       (s "(js/global.hasOwnProperty \"$$LUMO_GLOBALS\")

\ntrue
\n[1G[0Jcljs.user=> [13G"))
+;;   (string-match (concat form "\\Ca*true\\Ca*") s))
 
+(defun inf-clojure--lumo-mode-p (proc)
```

Comment:
I'd make this a generic function that takes as extra params some form and some expected result, so we could reuse it for other flavours.